koku/api/forecast/serializers.py

```diff
@@ -37,6 +38,38 @@ def validate(self, data):
 class AWSCostForecastParamSerializer(ForecastParamSerializer):
     """AWS Cost Forecast Serializer."""
 
+    COST_TYPE_CHOICE = (
```

Comment:
Just an observation, but this tuple is now defined in 2 different places. We ought to consider consolidating these to one place and importing where it is needed in a future PR.